.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "18.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
```

Comment:
I think you should check your linters set up to prevent your linters from unsuccessful checks. You can go through [our linters documentation](https://github.com/microverseinc/linters-config). to refresh your knowledge.